pkg/bundle/bundle_test.go

```diff
@@ -95,15 +111,17 @@ func Test_Reconcile(t *testing.T) {
 			},
 		}
 
-		baseBundleOwnerRef = []metav1.OwnerReference{*metav1.NewControllerRef(baseBundle, trustapi.SchemeGroupVersion.WithKind("Bundle"))}
+		baseBundleLabels = map[string]string{trustapi.BundleLabelKey: bundleName}
+
+		baseBundleOwnerRef = []metav1.OwnerReference{*metav1.NewControllerRef(baseBundle, trustapi.SchemeGroupVersion.WithKind(trustapi.BundleKind))}
 
 		namespaces = []client.Object{
 			&corev1.Namespace{TypeMeta: metav1.TypeMeta{Kind: "Namespace", APIVersion: "v1"}, ObjectMeta: metav1.ObjectMeta{Name: trustNamespace}},
 			&corev1.Namespace{TypeMeta: metav1.TypeMeta{Kind: "Namespace", APIVersion: "v1"}, ObjectMeta: metav1.ObjectMeta{Name: "ns-1"}},
 			&corev1.Namespace{TypeMeta: metav1.TypeMeta{Kind: "Namespace", APIVersion: "v1"}, ObjectMeta: metav1.ObjectMeta{Name: "ns-2"}},
 		}
 
-		fixedTime     = time.Date(2021, 01, 01, 01, 0, 0, 0, time.UTC)
+		fixedTime     = time.Date(2021, 01, 01, 01, 0, 0, 0, time.Local)
```

Comment:
question: Why change to Local? Does this not introduce the possibility of daylight savings weirdness?

I'd think that everything should be UTC unless there's a really good reason for it not to be!

Reply:
If you unmarshall metav1.Time from json, it automatically creates a Local time. (see https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/time.go#L115)
That is why I have to use Local time here, so I can compare this value after the bundel is marshalled and unmashalled in the test.

Reply:
Previously, we were comparing json. Now we are comparing go objects (using `assert.Equal`)

Reply:
Interesting; I wonder if tests should be run with the env var TZ set to ""? Hadn't really considered doing that before... https://pkg.go.dev/time#UTC

In any case, I can see why it's changed to `time.Local` here; not a blocker 😁